src/api/app/jobs/create_project_log_entry_job.rb

```diff
@@ -1,11 +1,7 @@
 class CreateProjectLogEntryJob < ApplicationJob
   queue_as :project_log_rotate
 
-  def perform(event_id)
-    event = Event::Base.find(event_id)
-    # If the ProjectLogEntry was invalid then we still mark the event as logged
-    event.update_attributes(project_logged: true)
-
-    ProjectLogEntry.create_from(event)
+  def perform(payload, timestamp, event_model_name)
```

Comment:
I would change `timestamp` => `created_at`, since timestamp can mean any kind of time but created_at tells us  what the variable represents.